js/store.js

```diff
@@ -68,7 +155,10 @@ const store = {
     const output = {};
     for (const hostname in this._websites) {
       const website = this._websites[hostname];
-      output[hostname] = this.outputWebsite(hostname, website);
+      // only previously ignored websites will have an 'isVisible' key
```

Comment:
Well, it's not possible in this version of the code. For some reason (which escapes me now), I felt I needed to have the key only exist on `thirdParty` if it was initially ignored. I will make the change to have the `isVisible` key always present for a `thirdParty` object.